group1/communication/node/main.js

```diff
@@ -29,27 +29,46 @@ var done = false;
 app.use(multer({
         'dest': '../../comm/static/uploads',
         onFileUploadStart: function(file) {
+            if(testNameValidation(file.originalname)){
```

Comment:
There should be a space between "if" and "(".  There should be another space between ")" and "{"